rrfs-test/validated_yamls/templates/obtype_config/atms_npp.yaml

```diff
@@ -0,0 +1,583 @@
+                                  0.050,  0.100]
+       # Calculate symmetric observation error
+       - filter: Variable Assignment
+         assignments:
+         - name: InitialObsError@DerivedMetaData
+           channels: *atms_npp_channels
+           type: float
+           function:
+             name: ObsErrorModelRamp@ObsFunction
+             channels: *atms_npp_channels
+             options:
+               channels: *atms_npp_channels
+               xvar:
+                 name: CLWRetSymmetric@DerivedMetaData
+               x0:    [ 0.030,  0.030,  0.030,  0.020,  0.030,
+                        0.080,  0.150,  0.000,  0.000,  0.000,
+                        0.000,  0.000,  0.000,  0.000,  0.000,
+                        0.020,  0.030,  0.030,  0.030,  0.030,
+                        0.050,  0.100]
+               x1:    [ 0.350,  0.380,  0.400,  0.450,  0.500,
+                        1.000,  1.000,  0.000,  0.000,  0.000,
+                        0.000,  0.000,  0.000,  0.000,  0.000,
+                        0.350,  0.500,  0.500,  0.500,  0.500,
+                        0.500,  0.500]
+               err0:  [ 4.500,  4.500,  4.500,  2.500,  0.550,
+                        0.300,  0.300,  0.400,  0.400,  0.400,
+                        0.450,  0.450,  0.550,  0.800,  4.000,
+                        4.000,  4.000,  3.500,  3.000,  3.000,
+                        3.000,  3.000]
+               err1:  [20.000, 25.000, 12.000,  7.000,  3.500,
+                        3.000,  0.800,  0.400,  0.400,  0.400,
+                        0.450,  0.450,  0.550,  0.800,  4.000,
+                       19.000, 30.000, 25.000, 16.500, 12.000,
+                        9.000,  6.500]
+
+       # Calculate Innovation@DerivedMetaData
+       - filter: Variable Assignment
+         assignments:
+         - name: Innovation@DerivedMetaData
+           channels: *atms_npp_channels
+           type: float
+           function:
+             name: ObsFunction/Arithmetic
+             channels: *atms_npp_channels
+             options:
+               variables:
+               - name: brightnessTemperature@ObsValue
+                 channels: *atms_npp_channels
+               - name: brightnessTemperature@HofX
+                 channels: *atms_npp_channels
+               coefs: [1, -1]
+
+       - filter: Bounds Check
+         filter variables:
+         #- name: brightnessTemperature
+         - name: HofX/brightnessTemperature
+           channels: *atms_npp_channels
+         minvalue: 50.00001
+         maxvalue: 449.99999
+         action:
+           name: reject
+
+       # Regional Domain Check
+       - filter: Bounds Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *atms_npp_channels
+         test variables:
+         - name: GeoVaLs/observable_domain_mask
+           flag all filter variables if any test variable is out of bounds: true
+           minvalue: 0.0
+           maxvalue: 0.5
```

Comment:
Do you know what the `flag all filter variables if any test variable is out of bounds` option does? That option is not enabled for our [mesonet yamls](https://github.com/NOAA-EMC/RDASApp/blob/bbfb811e56c2db7786db2b544ec530f82dba9fb1/rrfs-test/validated_yamls/templates/obtype_config/msonet_airTemperature_188.yaml#L63-L70). Also, for the mesonet yamls the `minvalue` and `maxvalue` keys are are indented under the main filter section and not under `test variables`. I am not sure if that matters though.

Reply:
1. "flag all filter variables if any test variable is out of bounds" can be removed here since only one test variable is involved.
2. I think minvalue and maxvalue keys should be listed under "test variables"

Reply:
Thanks @HuiLiu-NOAA for the input. For #&NoBreak;2, it seems like it worked for mesonet obs with the min/maxvalue indented under the filter and not under test variables. So maybe it works either way?

Reply:
I think it may be better to move minvalue and maxvalue keys under "test variables", just to be logically more clear.

Reply:
@xyzemc, @SamuelDegelia-NOAA, @HuiLiu-NOAA It looks like the max/min values keys are NOT indented in other filters in this yaml. For example, copied from step 3A. It seems to work either way? I have no issue with this as long as it works.
```yaml
       # Step 3A: CLW Retrieval Check (observation_based)
       - filter: Bounds Check
         filter variables:
         - name: brightnessTemperature
           channels: 1-7, 16-22
         test variables:
         - name: CLWRetFromObs@DerivedMetaData
         maxvalue: 999.0
         actions:
           - name: set
             flag: CLWRetrievalCheck
           - name: reject
```